src/icons/general/IconGlobe2.js

```diff
@@ -0,0 +1,34 @@
+// @flow
```

Comment:
does this need to be in box-ui-elements? imo we could hold off until its clear that it is general/foundational enough to be here.

Reply:
After talking with Lu, I am thinking the same thing. Closing this PR